src/new-components/button-link/button-link.stories.mdx

```diff
@@ -475,79 +487,92 @@ export function Template({ label, variant, tone, size }) {
         textLabel = 'Click me now'
     }
     return (
-        <Inline space="large">
-            <Stack space="large">
-                <Box>
-                    <ButtonLink variant={variant} tone={tone} size={size}>
-                        {label}
-                    </ButtonLink>
-                </Box>
-                <Box>
-                    <ButtonLink variant={variant} tone={tone} size={size} disabled>
-                        {label}
-                    </ButtonLink>
-                </Box>
-            </Stack>
-            <Stack space="large">
-                <Box>
-                    <ButtonLink variant={variant} tone={tone} size={size} startIcon={<Icon />}>
-                        {label}
-                    </ButtonLink>
-                </Box>
-                <Box>
-                    <ButtonLink
-                        variant={variant}
-                        tone={tone}
-                        size={size}
-                        startIcon={<Icon />}
-                        disabled
-                    >
-                        {label}
-                    </ButtonLink>
-                </Box>
-            </Stack>
-            <Stack space="large">
-                <Box>
-                    <ButtonLink variant={variant} tone={tone} size={size} endIcon={<Icon />}>
-                        {label}
-                    </ButtonLink>
-                </Box>
-                <Box>
-                    <ButtonLink
-                        variant={variant}
-                        tone={tone}
-                        size={size}
-                        endIcon={<Icon />}
-                        disabled
-                    >
-                        {label}
-                    </ButtonLink>
-                </Box>
-            </Stack>
-            <Stack space="large">
-                <Box>
-                    <ButtonLink
-                        variant={variant}
-                        tone={tone}
-                        size={size}
-                        aria-label={textLabel}
-                        tooltip={label}
-                        icon={<Icon />}
-                    />
-                </Box>
-                <Box>
-                    <ButtonLink
-                        variant={variant}
-                        tone={tone}
-                        size={size}
-                        aria-label={textLabel}
-                        tooltip={label}
-                        icon={<Icon />}
-                        disabled
-                    />
-                </Box>
-            </Stack>
-        </Inline>
+        <Stack space="large">
```

Comment:
For some reason the props table isn't rendering for the ButtonLink:

![image](https://user-images.githubusercontent.com/8531248/132839796-6513758c-2a36-4f38-9114-0abeb4f03d98.png)

Reply:
I tried exporting the component inline like in Tabs but it only managed to fix the canvas tab but not on the docs tab. Worth a fix anyway if it at least enables it for canvas:

|Before|After|
|-|-|
|![image](https://user-images.githubusercontent.com/8531248/132841268-007b1565-17e5-45ba-9d48-84b23d2d6dac.png)|![image](https://user-images.githubusercontent.com/8531248/132841232-bf18ad0b-bacd-4be5-bc10-a0eb761d6028.png)|

Reply:
I'll switch to inline export for the components affected in this PR.